templates/master.yaml

```diff
@@ -658,16 +547,16 @@ coreos:
     content: |
       [Unit]
       Description=k8s-api-server
-      Requires=calico-node.service
-      After=calico-node.service
+      Requires=calico-node.service k8s-key-generator.service
+      After=calico-node.service k8s-key-generator.service
+      StartLimitIntervalSec=0
 
       [Service]
       Restart=always
       RestartSec=0
       TimeoutStopSec=10
-      StartLimitIntervalSec=0
       EnvironmentFile=/etc/network-environment
-      Environment="IMAGE={{.Cluster.Kubernetes.Hyperkube.Docker.Image}}"
+      Environment="IMAGE={{.Spec.Docker.ImageNamespace}}/hyperkube:{{.Spec.Kubernetes.Hyperkube.Version}}"
```

Comment:
Path to that thing is probably much different now, let me find it later (or you can try to find faster :D )